src/util/assertUtil.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Assert that a given value is not `null` or `undefined`.
+ * @param dependency The value to assert.
+ * @param name The name of the value to include in the error message if the assertion fails.
+ */
+export function assertDefined<T>(
```

Comment:
Pretty much the same as `assertNotNull` in `@deephaven/utils`, not sure it's worth re-defining.